code/src/CoreTemplateStudio/CoreTemplateStudio.Core/Helpers/Fs.cs

```diff
@@ -18,9 +18,16 @@ public static class Fs
     {
         public static void EnsureFolder(string folder)
         {
-            if (!Directory.Exists(folder))
+            try
+            {
+                if (!Directory.Exists(folder))
+                {
+                    Directory.CreateDirectory(folder);
+                }
+            }
+            catch (Exception ex)
             {
-                Directory.CreateDirectory(folder);
+                AppHealth.Current.Error.TrackAsync($"Error creating folder {folder}: {ex.Message}", ex).FireAndForget();
```

Comment:
Saw other methods in the Fs Helper are tracking exceptions as warnings, we should follow the same pattern here